samples/previous-viewpoint/README.md

```diff
@@ -0,0 +1,3 @@
+# Go to previous viewpoint
+
+This sample demonstrates how to tap into a web event, and create a custom command for returning to the previous viewpoint.
```

Comment:
"Web" should be capitalized since it's the proper name of a product.

Reply:
Updated.